gamestonk_terminal/stocks/discovery/disc_controller.py

```diff
@@ -954,6 +954,17 @@ def call_fds(self, other_args: List[str]):
             help="Specify the Equities selection based on the description (not shown in table)",
         )
 
+        parser.add_argument(
+            "-mc",
+            "--marketcap",
```

Comment:
Also just caught this.  The convention we use is that single dashes gets a single letter, so it would be "-m"

Reply:
Changed it!